core/isolate.rs

```diff
@@ -556,13 +565,18 @@ impl Future for Isolate {
 
     loop {
       // If there are any pending dyn_import futures, do those first.
-      match self.pending_dyn_imports.poll() {
-        Err(()) => unreachable!(),
-        Ok(NotReady) => unreachable!(),
-        Ok(Ready(None)) => (),
-        Ok(Ready(Some((dyn_import_id, mod_id)))) => {
-          self.dyn_import_done(dyn_import_id, mod_id)?;
-          continue;
+      loop {
+        match self.pending_dyn_imports.poll() {
+          Ok(NotReady) | Ok(Ready(None)) => break,
+          Ok(Ready(Some((dyn_import_id, mod_id)))) => {
+            match self.mod_evaluate(mod_id) {
+              Ok(()) => self.dyn_import_done(dyn_import_id, Ok(mod_id))?,
+              Err(..) => self.dyn_import_done(dyn_import_id, Err(None))?,
```

Comment:
I’ve never seen this “..” before. That’s the same as “_” I guess ?

Reply:
practically speaking, yes